HealthcareSystem/PatientWebAppUnitTests/PatientControllerTests.cs

```diff
@@ -0,0 +1,117 @@
+using Backend.Service;
+using Model.Users;
+using Moq;
+using PatientWebApp.Controllers;
+using Repository;
+using System;
+using Xunit;
+using Model.Enums;
+using Microsoft.AspNetCore.Mvc;
+using Backend;
+using System.Collections.Generic;
+using PatientWebApp.DTOs;
+
+namespace PatientWebAppUnitTests
+{
```

Comment:
Is a good practice to use:
  - ProjectNameUnitTests
  - ProjectName.UnitTest
  - ProjectName.Tests

One is definitely, you make good prefix, **PatientWebApp**. We just need to make a convention in our organization. Convention about, **how to give a name to test projects**. And after that, everyone else should use that convention.

### My suggestion

Make this name for the project: **PatientWebApp.Tests**, after that, we can have a directory for **Unit tests** and a directory for **integration tests**.

Reply:
@sladjana-savkovic what do you think about this?

Reply:
@PSW-2020-ORG4/developers look at this convention when you make a unit test:
  - ProjectName
  - ProjectNameTest
    - Unit
    - Integration

Like in this image (for now there is the only unit, but when you make integration put them in one directory):

![image](https://user-images.githubusercontent.com/45834270/99889051-0bb0ab00-2c52-11eb-87bd-d9fcf7b7ec06.png)